### PR TITLE
Set sslmode=disable explicitly for plain Postgres

### LINKED ISSUE #1905: Non-ssl Postgres connection fails
Hey folks,

I'm trying to run Metabase against a Postgres installation that does not need/support SSL (because all traffic can be trusted.)  The docs say Metabase will prefer SSL but fall back to a regular connection, but that doesn't seem to be happening.

I grepped around for `ssl` and set random things to false, shooting in the dark, rebuilding the jar, but no luck.

### PR DESCRIPTION
**Pull request by dashkb**
closes https://github.com/metabase/metabase/issues/1905

@camsaul thanks for working with me on this!

### CODE REVIEW

File: src/metabase/driver/postgres.clj [hunk 1]
```diff
@@ -87,18 +87,22 @@
    :sslmode    "require"
    :sslfactory "org.postgresql.ssl.NonValidatingFactory"})  ; HACK Why enable SSL if we disable certificate validation?
 
+(def ^:const disable-ssl-params
+  "Params to include in the JDBC connection spec to disable SSL."
+  {:sslmode "disable" })
```
Comment (camsaul) [inline src/metabase/driver/postgres.clj]: minor nitpick: can you remove the extra space after `"disable"`?

Comment (dashkb) [inline src/metabase/driver/postgres.clj]: Of course!  That space sucks.

**Comment by camsaul** [comment]
@dashkb this looks good :+1: can you sign our [Contributor License Agreement](https://docs.google.com/a/metabase.com/forms/d/1oV38o7b9ONFSwuzwmERRMi9SYrhYeOrkbmNaq9pOJ_E/viewform)? Let me know when that's done and I'll merge this in

**Comment by dashkb** [comment]
> Let me know when that's done

It's done.

**Comment by camsaul** [comment]
:+1: Cool, I'll merge it as soon as the tests finish.

**Pull request merged**